stdlib/src/bit/bit.mojo

```diff
@@ -298,7 +302,7 @@ fn bit_width[
     else:
         var leading_zero_pos = countl_zero(val)
         var leading_zero_neg = countl_zero(bit_not(val))
-        var leading_zero = (val > 0).select(leading_zero_pos, leading_zero_neg)
+        var leading_zero = (val < 0).select(leading_zero_neg, leading_zero_pos)
```

Comment:
There was a small bug that cause the `bit_width` of 0 to be 'bitwidthof[type]'. This should fix it.

Reply:
thanks!